std_msgs/Int32 id
std_msgs/String subclass
std_msgs/String type
std_msgs/String name
VehicleParam param
bool ready_change_lane
State[] states